src/super_gradients/training/datasets/detection_datasets/detection_dataset.py

```diff
@@ -131,6 +134,15 @@ def __getitem__(self, index):
 
         return img, labels_out
 
+    @staticmethod
+    def mixup(im, labels, im2, labels2):
```

Comment:
@shaydeci - Don't we have mixup implemented already?

Reply:
I pulled Kate's PR which I had some comments on. This is mixup for detection as implemented in UL which is a bit different from what we have now for classification.